### DISCUSSION TITLE
Documentation about this display

### DISCUSSION THREAD

**Discussion by fr-fhe** [mathoudebine/turing-smart-screen-python#17] [Q&A]
Hello,

One of these screens is traveling to my home.
And i already know that i will thinker it !

Wondering if there is any documentation about communication protocol or if you just do reverse enginering on usb flow ?
What chipset inside ?

Dreaming of a complete ICD or datasheet :)

PS: native french people, but english typo to share worldwide

**Comment by mathoudebine** [comment]
Hello cher compatriote 😉 
Unfortunately i did not found any official documentation for the communication protocol or the hardware of the display.
This Python code has been written by decompiling the original Windows program (made in C# with Visual Studio) using JetBrains dotPeek.
If you find some docs i’m interested!

**Reply by fr-fhe** [reply]
Ok, so no miracle out there :)

Looking at your python code, command set seems easy : this is a blitter machine with RGB16 bitmaps.

After receiving my own device, I will write the java side of this library. 
Hope to have enough bravery to write little documentation on this serial API.

Did you do any benchmark on blitting speed ? (or any sustainable full screen fps ?).

We keep in touch !
